TabloidMVC/Repositories/PostRepository.cs

```diff
@@ -71,7 +72,7 @@ public Post GetPublishedPostById(int id)
                               LEFT JOIN Category c ON p.CategoryId = c.id
                               LEFT JOIN UserProfile u ON p.UserProfileId = u.id
                               LEFT JOIN UserType ut ON u.UserTypeId = ut.id
-                        WHERE IsApproved = 1 AND PublishDateTime < SYSDATETIME()
```

Comment:
When I add a post, IsApproved defaults to 1, so when it searches the postRepository in the Edit function (prev comment) this query doesn't return any posts